src/vendor_upstream/dom/DOMMouseMoveTracker.js

```diff
@@ -86,17 +86,20 @@ class DOMMouseMoveTracker {
       this._eventTouchStartToken = EventListener.listen(
         this._domNode,
         'touchstart',
-        this._onMouseMove
+        this._onMouseMove,
+        { passive: false }
       );
       this._eventTouchMoveToken = EventListener.listen(
-        this._domNode,
+        event.target,
         'touchmove',
-        this._onMouseMove
+        this._onMouseMove,
+        { passive: false }
       );
       this._eventTouchEndToken = EventListener.listen(
-        this._domNode,
+        event.target,
         'touchend',
-        this._onMouseUp
+        this._onMouseUp,
+        { passive: false }
```

Comment:
I'm registering these at the source element that triggered the touch (i.e, `event.target`) so we don't abruptly loose the touch handling just because the source element got removed.

TODO: Add a comment explaining this and link to moz docs.